modules/openapi-generator/src/main/resources/mysql-schema/mysql_schema.mustache

```diff
@@ -42,4 +42,108 @@ CREATE TABLE IF NOT EXISTS {{#defaultDatabaseName}}`{{{defaultDatabaseName}}}`.{
 {{/x-mysqlSchema}}
 {{/vendorExtensions}}
 
-{{/isArrayModel}}{{/hasVars}}{{/model}}{{/models}}
\ No newline at end of file
+{{/isArrayModel}}{{/hasVars}}{{/model}}{{/models}}
+{{#hasOAuthMethods}}
+--
+-- Table structure for table `oauth_clients`
+--
+CREATE TABLE IF NOT EXISTS `oauth_clients` (
+  `client_id`            VARCHAR(80)    NOT NULL,
+  `client_secret`        VARCHAR(80)    DEFAULT NULL,
+  `redirect_uri`         VARCHAR(2000)  DEFAULT NULL,
+  `grant_types`          VARCHAR(80)    DEFAULT NULL,
+  `scope`                VARCHAR(4000)  DEFAULT NULL,
+  `user_id`              VARCHAR(80)    DEFAULT NULL,
+  PRIMARY KEY (`client_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+--
+-- Table structure for table `oauth_access_tokens`
+--
+CREATE TABLE IF NOT EXISTS `oauth_access_tokens` (
+  `access_token`         VARCHAR(40)    NOT NULL,
+  `client_id`            VARCHAR(80)    DEFAULT NULL,
+  `user_id`              VARCHAR(80)    DEFAULT NULL,
+  `expires`              TIMESTAMP      NOT NULL,
+  `scope`                VARCHAR(4000)  DEFAULT NULL,
+  PRIMARY KEY (`access_token`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+--
+-- Table structure for table `oauth_authorization_codes`
+--
+CREATE TABLE IF NOT EXISTS `oauth_authorization_codes` (
+  `authorization_code`  VARCHAR(40)    NOT NULL,
+  `client_id`           VARCHAR(80)    DEFAULT NULL,
+  `user_id`             VARCHAR(80)    DEFAULT NULL,
+  `redirect_uri`        VARCHAR(2000)  NOT NULL,
+  `expires`             TIMESTAMP      NOT NULL,
+  `scope`               VARCHAR(4000)  DEFAULT NULL,
+  `id_token`            VARCHAR(1000)  DEFAULT NULL,
+  PRIMARY KEY (`authorization_code`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+--
+-- Table structure for table `oauth_refresh_tokens`
+--
+CREATE TABLE IF NOT EXISTS `oauth_refresh_tokens` (
+  `refresh_token`       VARCHAR(40)    NOT NULL,
+  `client_id`           VARCHAR(80)    DEFAULT NULL,
+  `user_id`             VARCHAR(80)    DEFAULT NULL,
+  `expires`             TIMESTAMP      on update CURRENT_TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `scope`               VARCHAR(4000)  DEFAULT NULL,
+  PRIMARY KEY (`refresh_token`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+--
+-- Table structure for table `oauth_users`
+--
+CREATE TABLE IF NOT EXISTS `oauth_users` (
+  `username`            VARCHAR(80)    DEFAULT NULL,
+  `password`            VARCHAR(80)    DEFAULT NULL,
```

Comment:
a password shouldn't be stored in the database. Really, we should have a hashed password or an encrypted blob

Reply:
Do you suggest to rename column into `password_hash` to make it more clear? I thought it's obvious for any anyone. I'm more worried about small column length (80).